downstream/assemblies/hub/assembly-managing-containers-hub.adoc

```diff
@@ -1,13 +1,12 @@
 ifdef::context[:parent-context: {context}]
 
 [id="managing-containers-hub"]
-= Managing containers in private automation hub
+= Managing containers in [PrivateHubName}
```

Comment:
This should be {PrivateHubName} but you have a square bracket in front which is breaking the attribute.